ga4gh/backends.py

```diff
@@ -365,18 +370,24 @@ class Backend(object):
     """
     Superclass of GA4GH protocol backends.
     """
-    def __init__(self, dataDir, Dataset):
+
+    def __init__(self, dataDir, variantSet):
```

Comment:
I would prefer `variantSetClass` here. To me this reads like the parameter is an *instance* of `VariantSet` not a *subclass* of `VariantSet`.